src/Jobs/QueueExport.php

```diff
@@ -72,6 +72,8 @@ public function handle(Writer $writer)
                 $sheet->open($sheetExport);
             }
 
+            $writer->raiseBeforeWritingEvent();
```

Comment:
It didn't raise these events before here, can you explain why it should be added there?

Reply:
I raised this event here because I noticed that this function is called only once and and calls the write() method. So I think I put this method call right before calling the write() method. Note that this is for queued exports only.